analyses/cell-type-wilms-tumor-06/notebook/07_combined_annotation_across_samples_exploration.Rmd

```diff
@@ -673,57 +633,48 @@ gain_profile <- wrap_plots(gain_profile, nrow = 1, widths = chromosome_arms$Size
-[Zheng et al, Front. Oncol (2023)](https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2023.1137346/full), 
-LOH at _chr1p_, _chr11p_ and _chr16q_ as well as _chr1q_ gain are prognostic factors.
+As described in [Nelson et al, Current Opinions in Pediatrics (2020)](https://pmc.ncbi.nlm.nih.gov/articles/PMC7919941/), [Phelps et al, Children (2018)](https://pmc.ncbi.nlm.nih.gov/articles/PMC6262554/) and [Zheng et al, Frontiers in Oncology (2023)](https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2023.1137346/full), LOH at *chr1p*, *chr11p* and *chr16q* as well as *chr1q* gain are prognostic factors.
 
-Here we aim to see if we identify CNV associated with poorer prognosis. 
+Here we aim to see if we identify CNV associated with poorer prognosis.
 
 ```{r fig.height=15, fig.width=20, message=FALSE, warning=FALSE, out.width='100%', results='asis'}
 do_BoxPlot_mean(df = cell_type_df_sub, group.by = "vital_status", comparison = c("Alive", "Expired"))
 ```
 
-In our study, we validate the association of _chr1q_ gain, _chr11q_ loss and _chr14q_ loss with poorer prognosis.
+In our study, we validate the association of *chr1q* gain, *chr11q* loss and *chr14q* loss with poorer prognosis.
 
 ##### `treatment`
 
-As we don't know the effect of the treatment on the selection of cells with CNV, it is difficult to compare our CNV profile with the one in [Cresswell et al, BioRXiv (2024)](https://www.biorxiv.org/content/10.1101/2024.09.04.610994v1).
+As we don't know the effect of the treatment on the selection of cells with CNV, it is difficult to compare our CNV profile with the one in [Cresswell et al, bioRxiv (2024)](https://www.biorxiv.org/content/10.1101/2024.09.04.610994v1). 
 We those check the repartition of CNV in samples that have been pre-treated (i.e `Resection post chemotherapy`) with samples that have been removed surgically before any treatment (i.e `Upfront resection`).
 
-
 ```{r fig.height=15, fig.width=20, message=FALSE, warning=FALSE, out.width='100%', results='asis'}
 do_BoxPlot_mean(df = cell_type_df_sub, group.by = "treatment", comparison = c("Upfront resection", "Resection post chemotherapy"))
 ```
 
-Our analysis suggest that there is indeed an enrichment in cells with CNV after chemotherapy, 
-including a tendency of more _chr1q_ gain samples after chemotherapy. 
-This could explain some of the differences observed between our mean CNV profile and the one published in [Cresswell et al, BioRXiv (2024)](https://www.biorxiv.org/content/10.1101/2024.09.04.610994v1).
-
+Our analysis suggest that there is indeed an enrichment in cells with CNV after chemotherapy, including a tendency of more *chr1q* gain samples after chemotherapy. This could explain some of the differences observed between our mean CNV profile and the one published in [Cresswell et al, bioRxiv (2024)](https://www.biorxiv.org/content/10.1101/2024.09.04.610994v1).
```

Comment:
```suggestion
Our analysis suggest that there is indeed an enrichment in cells with CNV after chemotherapy, including a tendency of more *chr1q* gain samples after chemotherapy. This could explain some of the differences observed between our mean CNV profile and the one published in [Cresswell et al. (2024)](https://www.biorxiv.org/content/10.1101/2024.09.04.610994v1).
```